src/devices/Adxl345/Adxl345.cs

```diff
@@ -34,89 +35,115 @@ public class Adxl345 : IDisposable
         private const byte ACCEL_SPI_MB_BIT = 0x40;         // Bit used to indicate multi-byte SPI transactions    
         #endregion
 
+        /// <summary>
+        /// Read Acceleration from ADXL345
+        /// </summary>
+        public Vector3 Acceleration => ReadAcceleration();
+
         /// <summary>
         /// Constructor
         /// </summary>
-        /// <param name="os">The program runing platform (Linux or Windows10)</param>
         /// <param name="busId">SPI Bus ID</param>
-        /// <param name="chipSelect">CS Port</param>
-        /// <param name="gravityRange">Gravity Range</param>
-        public Adxl345(OSPlatform os, int busId, int chipSelect, GravityRange gravityRange)
+        /// <param name="chipSelect">CS Pin</param>
+        /// <param name="gravityRange">Gravity Measurement Range</param>
+        public Adxl345(int busId, int chipSelect, GravityRange gravityRange)
         {
-            _busId = busId;
-            _os = os;
-            _cs = chipSelect;
-
-            if (gravityRange == GravityRange.Two)
+            if (gravityRange == GravityRange.Range1)
             {
                 _range = 4;
             }
-            else if (gravityRange == GravityRange.Four)
+            else if (gravityRange == GravityRange.Range2)
```

Comment:
I didn't have any issue with previous name, just internal value (i.e. instead of Four = 2 use Four = 8, so that you can use it directly here instead if long if-else-if... block)

Reply:
enum **GravityRange** internal value means register address, and **_range** is the variable I used to calculate the unit, I need to convert the enum to _range😁